src/main/java/com/dev/movietheatre/model/CinemaHall.java

```diff
@@ -0,0 +1,48 @@
+package com.dev.movietheatre.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class CinemaHall {
```

Comment:
add `@Table` annotation and configure `proper_table_name`

Reply:
done